src/ui/page/notFoundError/i18n/notFoundError_fr.json

```diff
@@ -0,0 +1,8 @@
+{
+  "notFoundError": {
+    "apology": "Désolé,",
+    "message": "Nous ne trouvons pas la page que vous avez demandée .",
```

Comment:
```suggestion
    "message": "Nous ne trouvons pas la page que vous avez demandée.",
```

Reply:
I thought that's how French punctuation works xD Fixed now!